doc/abrt-oops.conf.txt

```diff
@@ -14,13 +14,17 @@ DESCRIPTION
 The configuration file consists of items in the format "Option = Value".
 The following items are recognized:
 
-DropNotReportableOopses = 'yes' / 'no'
-   If you want to see only reportable oopses, set to "yes".
-   Default is 'no': do not drop them.
-
-OnlyFatalMCE = 'yes' / 'no'
-   If you want to see only fatal MCEs, set to "yes".
-   Defaults is 'yes': detect only fatal ones.
+*DropNotReportableOopses = 'yes/no'*::
+   If enabled, only reportable oopses with useful backtraces will be preserved.
```

Comment:
The backtrace can be useful but the oops might come from a tainted kernel, which makes it not reportable.

Reply:
I'm not sure I'm following. Where is this logic implemented? I only checked `a-a-analyze-oops` and the messages don't mention tainting either.

https://github.com/abrt/abrt/blob/f33b873/src/plugins/abrt-action-analyze-oops.c#L64-L96

Reply:
https://github.com/abrt/abrt/blob/eb986de7b2de0f7724fdcdce74670a2e0ea75a56/src/plugins/oops-utils.c#L236

Reply:
`If enabled, ABRT will preserve only reportable oopses.` ?